OPI_RELEASE_APPROACH.md

```diff
@@ -0,0 +1,42 @@
+# OPI Release Approach
+
+OPI will eventually release some artifacts around the work being done in the
+project. What those artifacts are is still being discussed. However, *how*
```

Comment:
I don't like the `still being discussed` language, we need to decide first and then merge PR

Reply:
Fixed.